src/installer/corehost/cli/test/nativehost/host_context_test.cpp

```diff
@@ -380,6 +420,91 @@ namespace
 
         return rc == StatusCode::Success && rcClose == StatusCode::Success;
     }
+
+    bool get_function_pointer_test(
```

Comment:
I would rename this to something like `component_get_function_pointer_test` to make it clear that this is on the "component" context.